src/profile/helpers/editData.ts

```diff
@@ -80,17 +87,42 @@ export type EditData = {
   readonly saving: SaveType;
 };
 
+type Backups = {
+  add: (item: EditData) => void;
+  clean: (allItems: EditData[]) => void;
+  get: (id: string) => EditData | undefined;
+};
+
 export type EditFunctions = {
   create: (newProfileData: EditDataProfileSource) => EditData;
   getEditData: () => EditData[];
+  updateItemAndCreateSaveData: (
+    targetItem: EditData,
+    newValue: EditDataValue
+  ) => Partial<FormValues>;
+  updateData: (newProfileRoot: ProfileRoot) => boolean;
+  updateAfterSavingError: (id: string) => boolean;
+  resetItem: (targetItem: EditData) => boolean;
 };
 
-function isMultiItemDataType(dataType: EditDataType): boolean {
+export function isMultiItemDataType(dataType: EditDataType): boolean {
```

Comment:
Is this export necessary? Tried searching but did not find any usages outside of this file?

Reply:
Removed